### PR TITLE
Update spanish translations

### PR DESCRIPTION
**Pull request by JonathonSonesen**
This is my first time contributing to a project, I am a computer science student at Portland Community College, a friend of mine is an employee at discogs and advised me that I could start plugging away at translations so here I am!

### CODE REVIEW

**Comment by JonathonSonesen** [comment]
made a mistake on commit 09a9b91b

File: es/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -46,15 +46,15 @@ msgid ""
...
 
 #, python-format
 msgid "\"%s\" is available."
-msgstr ""
+msgstr "\"%s\" esta disponible"
 
 #, python-format
 msgid "\"Search %(seller)s's store"
-msgstr ""
+msgstr "\"Buscar tienda de %(seller) "
```
Comment (cburmeister) [inline es/LC_MESSAGES/messages.po]: That last `s` will cause a syntax error. The `%(whatever)s` chunks are placeholders.

**Comment by JonathonSonesen** [comment]
Yeah I realized that after committing the change

File: es/LC_MESSAGES/messages.po (hunk 1 above)
Comment (JonathonSonesen) [inline es/LC_MESSAGES/messages.po]: Do you mean that having the last s missing? I realized that was an error after submitting the pull

**Pull request closed without merging**